### DISCUSSION TITLE
Performance comparison between System.Data.SqlClient and Microsoft.Data.SqlClient

### DISCUSSION THREAD

**Discussion by qizu0302** [dotnet/SqlClient#1581] [Q&A] [answered]
Hi SQL experts, I wonder performance comparison between System.Data.SqlClient and Microsoft.Data.SqlClient. 
Is there degradation or upgrade?

**Comment by johnnypham** [comment, marked as answer]
I don't think there has been any rigorous benchmarking done but maybe I'm wrong. A lot of minor perf improvements have been made to Microsoft.Data.SqlClient and going forward it's the recommended library.
